x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/AbstractScalarFunctionTestCase.java

```diff
@@ -304,11 +305,17 @@ public final void testEvaluateInManyThreads() throws ExecutionException, Interru
         if (testCase.getExpectedBuildEvaluatorWarnings() != null) {
             assertWarnings(testCase.getExpectedBuildEvaluatorWarnings());
         }
+
+        List<Object> simpleData = testCase.getDataValues();
+        // Ensure we don't run this test with too much data that could take too long to process.
+        // The calculation "ramUsed * count" is just a hint of how much data will the function process,
+        // and the limit is arbitrary
+        assumeTrue("Input data too big", row(simpleData).ramBytesUsedByBlocks() * count < GB.toBytes(1));
```

Comment:
Why's this needed? Did we make REPLACE super slow?

Reply:
As I added the test with a lot of data to the ReplaceTests file, this specific case was very slow (3s -> 40s).

Anyway, I just moved the big cases to another file, so they're executed just once. Same as `RepeatStaticTests` does. So this isn't needed anymore